src/xdp/conntrack_common.h

```diff
@@ -93,6 +94,55 @@ static inline int enforce_ingress_policy(__u64 tunnel_id, const struct ipv4_tupl
 
 	if (0 == policies_ppo) return -1;
 
+	if (pod_label_value > 0) {
+		bpf_debug("Checking packet for pod label policy with pod_label_value=%d",
```

Comment:
connection tracking map only includes ip address. Can you explain the loggic over here? How are you applying label to connection tracking map?

Reply:
discussed with hong offline. resolved